Source/UI/NewUI/CommonUI/View/Core/ImageContainerView.swift

```diff
@@ -20,7 +20,7 @@ class ImageContainerView: UIView {
     }
 
     func update(with image: UIImage?, tintColor: UIColor? = nil) {
-        imageView?.image = image
+        imageView?.image = image?.withRenderingMode(.alwaysTemplate)
```

Comment:
Question : Did you not need a new image form Alex our designer ? or you are just changing the tint color

Reply:
just changing the tint color

Reply:
can we make this only with `.withRenderingMode(.alwaysTemplate)` if the `tintColor` is not nil? This current change will break card scheme icons once they are PDFs

Reply:
```suggestion
        imageView?.image = tintColor == nil ? image ? image?.withRenderingMode(.alwaysTemplate)
```